emark/registry.py

```diff
@@ -0,0 +1,32 @@
+from django.urls import reverse_lazy
+
+
+class EmarkRegistry:
+    def __init__(self):
+        self.emails = {}
+
+    def __iter__(self):
+        return iter(self.emails.values())
+
+    def __call__(self, email_class):
+        self.emails[email_class.__name__] = {
+            "app_label": email_class.__module__.split(".")[0],
+            "class": email_class,
+            "name": email_class.__name__,
+            "doc": email_class.__doc__ or "",
```

Comment:
I don't really see where most of those are being used.
Why not just hold a reference to the class? If no explicit key for the

Reply:
All these are used in the templates and I cannot (also don't want to) access dunder methods there.
And there is a bit of logic to get `app_label` (which I am using for grouping the emails under their respectful apps).

I'd be happy to get a better idea on how to arrange all this.